lib/tasks/statistics/load_temperatures.rake

```diff
@@ -0,0 +1,56 @@
+namespace :statistics do
+  desc "Get benchmarks for temperature data loads"
+  task load_temperatures: :environment do
+    weather_settings = School.all.map { |school| { dark_sky_area_id: school.dark_sky_area_id, weather_station_id: school.weather_station_id } }.uniq
+    benchmarks = [] # Store all benchmark data for later output to csv
+    weather_settings.each do |weather_ids|
+      weather_station_id = weather_ids[:weather_station_id]
+      dark_sky_area_id = weather_ids[:dark_sky_area_id]
+      weather_station_title = weather_station_id ? WeatherStation.find(weather_station_id)&.title : 'none'
+      dark_sky_area_title = dark_sky_area_id ? DarkSkyArea.find(dark_sky_area_id)&.title : 'none'
+
+      puts "\nBenchmark for weather_station: #{weather_station_title} (#{weather_station_id || 'nil'}) and dark_sky_area: #{dark_sky_area_title} (#{dark_sky_area_id || 'nil'})"
+
+      temperatures = Temperatures.new('temperatures')
```

Comment:
I think this should be within the benchmarking code, otherwise repeated runs are manipulating the same object?

Reply:
:+1: done